core/commands/unixfs/ls.go

```diff
@@ -64,6 +66,8 @@ Example:
 
 This functionality is deprecated, and will be removed in future versions. If
```

Comment:
Let's try to keep this as concise as possible:

"...in future versions _as it duplicates the functionality of `ipfs ls`_."

We can then drop the other changes.